benchmarks/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+description = "A module containing benchmarks for the other modules"
+
+plugins {
+    kotlin("multiplatform")
+    id("kotlinx.benchmark") version Versions.kotlinBenchmark
+    id("org.jetbrains.kotlin.plugin.allopen") version Versions.kotlin
+}
+
+allOpen {
+    annotation("org.openjdk.jmh.annotations.State")
+    annotation("org.openjdk.jmh.annotations.BenchmarkMode")
+}
+
+sourceSets.create("benchmark")
+
+benchmark {
+    targets {
+        register("jvmBenchmark")
```

Comment:
so at the time of converting to mpp, we only targeted jvm. But now there is a js target, and as we can see from the benchmark plugin description: https://github.com/Kotlin/kotlinx-benchmark
this should work for javascript too.

could we reorganize and move the benchmark code into `commonBenchmark`? 
I dont believe there is really anything in the current benchmark file that is jvm specific.

They have a great example for a multiplatform benchmark in the plugin repo, 
we can see the imports we would need to change here to make them multiplatform compatible: https://github.com/Kotlin/kotlinx-benchmark/blob/master/examples/kotlin-multiplatform/src/commonMain/kotlin/CommonBenchmark.kt

the build script for that example is also helpful: https://github.com/Kotlin/kotlinx-benchmark/blob/master/examples/kotlin-multiplatform/build.gradle